Makefile

```diff
@@ -45,6 +45,7 @@ deploy-lib-jitsi-meet:
 		$(LIBJITSIMEET_DIR)/browser_capabilities.min.js \
 		$(LIBJITSIMEET_DIR)/browser_capabilities.min.map \
 		$(LIBJITSIMEET_DIR)/connection_optimization/external_connect.js \
+		$(LIBJITSIMEET_DIR)/modules/browser/capabilities.json \
```

Comment:
Are we going to keep building browser_capabilities.min.{js,map} in lib-jitsi-meet? I think 'cp' will break if the files don't exist.